### DISCUSSION TITLE
After Restarting containers builder/admin redirect

### DISCUSSION THREAD

**Discussion by juliansworkshop** [Budibase/budibase#2591] [Help] [answered]
I have built a few apps and they have already been put into use after running into a condition in the app builder that had seemingly caused a memory leak I restarted all the budibase containers using docker restart everything seemed to work fine again apps were functioning again and everything looked fine.
Few hours later my session expired and instead of redirecting me to the login page its redirecting me to Create admin user page even if I manually try to go to the login page. Further it fails to create a new superuser since there's already one and I am not easily finding a way to force it out of this initializing phase.

![users](https://user-images.githubusercontent.com/54377525/132978394-d211a169-740e-4c3e-b138-5bdd50750099.jpg)

The users are clearly still in the database but I cant get to the login page to login with them.

I was able to turn these services on and off in the past while testing and it did not go back to this initial state during then. The only thing I could think of that might have triggered it was I clicked the check for update button within the ui figuring it would see if there was an update not just go and do it.

My yml is basically unchanged from the default other than some labels setting the network and removing some external port mappings. 

I notice this in the logs right before it redirects to the /builder/admin page
  error: 'not_found',
  reason: 'Database does not exist.',
  status: 404,
  name: 'not_found',
  message: 'Database does not exist.',
  docId: 'app_metadata'

**Comment by shogunpurple** [comment, marked as answer]
Hi @juliansworkshop, thank you very much for the very detailed and well written discussion. It's great to hear that you have built a few apps with budibase and are using them!

I think the problem here is an outdated `envoy.yaml` file, which we use for our proxy service - a service that sits in front of all the budibase services and routes traffic to them based on path.

To update your `envoy.yaml` to the latest, navigate to the directory of your budibase installation containing `docker-compose.yaml` etc and run:

```
mv envoy.yaml envoy.yaml.old
wget https://github.com/Budibase/budibase/raw/develop/hosting/envoy.yaml
docker-compose restart proxy-service
```
This should hopefully resolve the issues you are seeing. Let me know if this helps!

**Reply by juliansworkshop** [reply]
Thanx just a side note the formatting on the new one is off was getting error at line 40, column 1: end of map not found cleaned up some white space and it worked.
Wish I was more patient and didn't try solve this myself I must've broken everything and fixed it again like 3 times hopefully Ill use couch db again for something.
